standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/dataconnector/jdbc/MySQLConnectorProvider.java

```diff
@@ -31,7 +31,7 @@
 public class MySQLConnectorProvider extends AbstractJDBCConnectorProvider {
   private static Logger LOG = LoggerFactory.getLogger(MySQLConnectorProvider.class);
 
-  private static final String DRIVER_CLASS = "org.mariadb.jdbc.Driver";
+  private static final String DRIVER_CLASS = "com.mysql.jdbc.Driver";
```

Comment:
I noticed this too yesterday and was wondering about this. This was change in https://issues.apache.org/jira/browse/HIVE-25423. I am wondering if this change back would break the test infra. Have you given it a try? Thanks

Reply:
Yes， I have tested the pr with local hive cluster and mysql.  I also have  run qtest `dataconnector_mysql.q` which was added in HIVE-25423.
I'm not sure why would HIVE-25423 change DRIVER_CLASS to mariadb. Anyway, if i change it back to mysql driver, my hive cluster will be ok.

Reply:
I am not sure why HIVE-25423 changed the driver to use mariadb but maybe it has to do with the fact that the added test (dataconnector_mysql.q) was also using a docker image of MariaDB (even though the respective class was implying that it was MySQL) before I fix it on HIVE-25594.

Reply:
Thats was the exact goal for this framework as well. 

Are there any new connectors you plan on adding or have added that can be contributed back to the hive community? or any bug fixes you may have made? (I recently opened a bug for the any documentation enhancements in the Apache docs would be very helpful as well.

Reply:
@zhangbutao Given your extensive work with MySQL, would you be able to look at https://issues.apache.org/jira/browse/HIVE-26046 ? its about mapping bit datatypes from mysql to hive.

Reply:
> Are there any new connectors you plan on adding or have added that can be contributed back to the hive community? or any bug fixes you may have made? (I recently opened a bug for the any documentation enhancements in the Apache docs would be very helpful as well.

@nrg4878 Currently  we mostly use existing mysql jdbc data connectors. If we add a new data connecotr, I'm more than happy to contribute it to the community.  I will explore storage handler and data connector in depth, and i will fire pr if finding related bugs. :)

>  Given your extensive work with MySQL, would you be able to look at https://issues.apache.org/jira/browse/HIVE-26046 ? its about mapping bit datatypes from mysql to hive.

Sure, I'd be happy to look into HIVE-26046. I will give a fix as soon as possible if i can solve it.
Thanks!